app/src/main/java/com/powerhungers/fomezero/feature/registration/presentation/RegistrationFragment.kt

```diff
@@ -43,5 +61,19 @@ class RegistrationFragment : Fragment() {
     }
 
     private fun addObserver() {
+        viewModel.registrationLiveData.observe(viewLifecycleOwner){state ->
+            with(binding){
+                when(state){
+                    is ViewState.Loading ->
+                        progressDialog.isVisible = true
+                    is ViewState.Success ->{
+                        progressDialog.isGone = true
+                    }
+                    is ViewState.Error ->
+                        progressDialog.isGone = true
+                    else -> {Unit}
+                }
+            }
+        }
```

Comment:
```suggestion
        viewModel.registrationLiveData.observe(viewLifecycleOwner) { state ->
            with(binding) {
                when(state) {
                    is ViewState.Loading ->
                        progressDialog.isVisible = true
                    is ViewState.Success -> {
                        progressDialog.isGone = true
                    }
                    is ViewState.Error ->
                        progressDialog.isGone = true
                    else -> {Unit}
                }
            }
        }
```
indentação incorreta